master_changes.md

```diff
@@ -32,6 +32,9 @@ users)
 
 ## Remove
   *
+  * Fix removal of root packages with `-a` and an optional dependency explicitely specified [@AltGr - fix #4727]
+  * Fix message when running `opam remove` on an unavailable package [@AltGr - fix #4890]
+  * Fix removal of root packages with `-a` and an optional dependency explicitely specified [@AltGr - fix #4727]
```

Comment:
these 3 changelog entries seem like the perfect candidates to translate into 3 minimal tests to add to the testsuite

Reply:
Fair enough...